backend/internal/domain/services/task_list_service.go

```diff
@@ -0,0 +1,195 @@
+package services
+
+import (
+	"context"
+	"net/http"
+	"shodo/internal/domain/helpers"
+	"shodo/internal/repository"
+	"shodo/models"
+)
+
+const (
+	kDefaultTaskListTitle = "Shoppings"
+	kNotAllowed           = "operation permitted only for list owner or shared users"
+)
+
+type TaskListService struct {
+	TaskListRepository    repository.TaskList
+	UsersRepository       repository.Users
+	AuthenticationService *AuthenticationService
+}
+
+func (this *TaskListService) CreateDefaultTaskList(ctx context.Context, username, ownerId string) {
+	this.TaskListRepository.CreateTaskList(ctx, &models.TaskList{Title: username + " " + kDefaultTaskListTitle, Owner: ownerId})
+}
+
+func (this *TaskListService) CreateTaskList(ctx context.Context, list *models.TaskList) {
+	this.TaskListRepository.CreateTaskList(ctx, list)
+}
+
+func (this *TaskListService) GetTaskLists(ctx context.Context, userToken string) ([]models.TaskListShort, error) {
+	userId, err := helpers.GetUserIdFromToken(userToken)
+	if err != nil {
+		return nil, err
+	}
+
+	return this.TaskListRepository.GetTaskLists(ctx, userId)
+}
+
+func (this *TaskListService) GetTaskList(ctx context.Context, listId *string, userToken string) (models.TaskList, *models.Error) {
+	isListExists, err := this.TaskListRepository.CheckTaskListExists(ctx, *listId)
```

Comment:
You rarely use is... in front of a boolean.
`exists` would be a better name and it reads well

`if exists` is nicer than `if isListExists`
Also, you usually don't put the datatype name in a variable

You already know the type since this is a strongly typed language